src/Store/SyntaxHighlightingStoreConnector.re

```diff
@@ -33,14 +33,9 @@ let start = (~enabled, languageInfo: Ext.LanguageInfo.t) => {
        );
   };
 
-  let mapServiceEffect:
-    Isolinear.Effect.t(Service_Syntax.msg) =>
-    Isolinear.Effect.t(Model.Actions.t) =
-    effect =>
-      Isolinear.Effect.map(
-        msg => {Model.Actions.Syntax(Feature_Syntax.Service(msg))},
-        effect,
-      );
+  let mapUnitEffect:
+    Isolinear.Effect.t(unit) => Isolinear.Effect.t(Model.Actions.t) =
+    effect => Isolinear.Effect.map(() => Model.Actions.Nothing, effect);
```

Comment:
This whole file needs to be cleaned up - the 'eager' syntax highlighting might be able to be brought over to `Feature_Syntax` now. If that's the case, the `SyntaxHighlightingStoreConnector` can go away.

Reply:
`SyntaxHighlightingStoreConnector` is now removed